Add showDaysOnly option and translations

CristianCantoro opened the pull request:
As per the commit title, this pull request adds two new options:
* `showDaysOnly` that, if set to `true` shows the differences only up to days
* `labelsTranslations`, a dictionary that can be used to translate the labels (`year`, `month`, `day`. `hour`, `minute`, `second`)
```javascript
$('#counter-days-only').countdownCube( {
  target: new Date( 'March 1, 2018 10:00:00' ),
  cubeSize: 150,
  background:  '#ffffff',
  color: 'blue',
  labelsTranslations: {'year': 'anni',
                       'month': 'mesi',
                       'day': 'giorni',
                       'hour': 'ore',
                       'minute': 'minuti',
                       'second': 'secondi'
                       },
  showDaysOnly: true,
  } );
```

CristianCantoro committed:
Add showDaysOnly option and translations

jgb-solutions commented:
nice!

oofaish commented:
Hey - amazing work Cristian - can I just make 1 request before we merge? Can you remove the labelsTranslation from one of the examples so people see they dont have to set that property? or just add a line in the demo saying so? but nice work. ❤️

CristianCantoro committed:
Fix defaults for translations, show JS code in the demo page

CristianCantoro commented:
@oofaish said:
> can I just make 1 request before we merge? Can you remove the labelsTranslation from one of the examples so people see they dont have to set that property?

Great suggestion, while doing that I realized that I did not add a default for the labels, now it's done (and fixed). I have also updated the demo page and I believe is more clear now.

I have another pull request in the making :)
